.editorconfig

```diff
@@ -11,3 +11,7 @@ insert_final_newline = true
 
 [*.md]
 trim_trailing_whitespace = false
+
+[*.yml]
+indent_style = space
+indent_size = 2
```

Comment:
All the YAML files were using spaces, so it seemed appropriate to update the `.editorconfig` accordingly.